t/plugin/google-cloud-logging.t

```diff
@@ -793,9 +850,90 @@ passed
 
 
 
-=== TEST 27: test route (set include_req_body = true on route succeeds)
+=== TEST 28: set fetch request body and response body route
+--- config
+    location /t {
+        content_by_lua_block {
+            local config = {
+                uri = "/google-cloud-logging/test",
+                method = 'POST',
+                upstream = {
+                    type = "roundrobin",
+                    nodes = {
+                        ["127.0.0.1:12001"] = 1
+                    }
+                },
+                plugins = {
+                    ["google-cloud-logging"] = {
+                        auth_file = "t/plugin/google-cloud-logging/config.json",
+                        inactive_timeout = 1,
+                        batch_max_size = 1,
+                        include_req_body = true,
+                    }
+                }
+            }
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1', ngx.HTTP_PUT, config)
+
+            if code >= 300 then
+                ngx.status = code
+                ngx.say(body)
+                return
+            end
+
+            ngx.say(body)
+        }
+    }
+--- response_body
+passed
+
+
+
+=== TEST 28: set fetch request body and response body route
```

Comment:
The sequence number is wrong? It should be `29`.

Reply:
@tokers I removed the test 26 & 27 I added in an initial commit while trying to get tests working and the test 28 & 29 referenced above become 26 & 27.